ckb-importer/src/main/java/com/hartwig/serve/ckb/datamodel/treatmentapproaches/RelevantTreatmentApproachesFactory.java

```diff
@@ -36,8 +35,8 @@ private static RelevantTreatmentApproaches resolveRelevantTreatmentApproaches(@N
             if (treatmentApproach.id() == treatmentApproachInfo.id()) {
                 return ImmutableRelevantTreatmentApproaches.builder()
                         .id(treatmentApproach.id())
-                        .drugClass(treatmentApproach.drugClass() != null ? DrugFactory.resolveDrugClass(ckbJsonDatabase,
-                                treatmentApproach.drugClass()) : null)
+                        .drugClass(treatmentApproach.drugClass() != null ? treatmentApproach.drugClass() : null)
+                        .therapy(treatmentApproach.therapy() != null ? treatmentApproach.therapy() : null)
```

Comment:
The ternary operator is redundant here:
```suggestion
                        .drugClass(treatmentApproach.drugClass())
                        .therapy(treatmentApproach.therapy())
```

Reply:
implemented your suggestion